free-1.md

```diff
@@ -6,63 +6,92 @@ good way to meet their needs.
+file_not_found errors. Perhaps I could create an error checker for this.
 
 
 ### Expressiveness
-_What should be easy to do in this language? What should be possible, but
-difficult? What should be impossible or very difficult?_
+It should be easy to create a simple dynamic web page with the language. 
+It would be difficult to put anything on the website that you cannot put
+in a Word document. Anything besides making a website would most likely
+be impossible. 
 
 
 ### Related work
-_Are there any other DSLs in this domain? If not, describe how you know there
-aren't and conjecture why not. If so, describe them and provide links. How well 
-do they address the need? Are there any particularly admirable qualities of the
-language? Are there parts of the language you think could be improved?_
+I do not think there are any other DSLs in this specific language however, 
+there are tools that address the need that I am trying to address. Programs
+such as Wordpress allow non-technical users to create personal websites. There
+are also an almost unlimited number of websites on which you can create a personal
+site. A few are listed below.
 
++ https://wordpress.com
++ http://www.wix.com
++ http://www.squarespace.com
++ http://www.weebly.com
 
 ## The Project
 This section examines whether the idea makes for a good CS 111 project.
 
 
 ### Suitability
-_If someone were to work on this project, what percentage of their time would be
-spent directly engaging in the **language** aspects of this project (e.g.,
-making language design decisions), as opposed to "systems" aspects of the
-project (e.g., implementing a complicated semantics that doesn't require a lot
-of language design)?_
-
+I would think that the project would be more of working on the system of the 
+language rather than language design, although I do not see a shortage of work
+for language design. 
 
 ### Scope
-_How big an idea is this? How ambitious is this project?_
+This project is somewhat ambitious. I don't know how to parse word documents. I 
+do know that there are libraries in python for writing to Excel files so that 
+part would not be difficult. Also Word already has an option for turning a
+document into an HTML file so that part would not be difficult. 
 
 
 ### Benefits and drawbacks
-_Why might this be a good idea for a project? Why might this not be a good idea 
-project?_
+This would be a good idea for a project because it seems like the right amount
+of work and also seems like a fun language to implement. I would be implementing
+for users on the opposite end of the technical spectrum than myself so it would
+realy make me think from a different perspective. I also see this as a good way
+to get Microsoft Office users introduced to the ideas of programming. A SQL 
+database in essence is the same as an Excel spreadsheet and Word document is the
+same as an HTML file. Perhaps if users saw it from that perspective, they could 
```

Comment:
"the same" as in analogous, or do you mean the fact that Word docs are XML, or what?